.github/workflows/test.yml

```diff
@@ -9,14 +9,18 @@ on:
       os-packages:
         required: false
         type: string
+      py-versions:
+        required: false
+        default: '["3.11", "3.10", "3.9", "3.8"]'
```

Comment:
Oh, right 3.12 is also supported on 6.0. We are not using it on Jenkins though, we do 3.8 and 3.11 (and .9 and .10 on a weekly basis).

Since this configuration here is to run only package tests, I thought it would make sense to run all versions, as they would not take long, but indeed, ♻️ let's try to be eco-friendly and only run min and max also on GHA 👍🏾

Reply:
@gforcada i have configured and tested my dymmy repro with the branch `multiple-pythons`. LGTM.